Library/Homebrew/github_packages.rb

```diff
@@ -239,8 +239,8 @@ def upload_bottle(user, token, skopeo, formula_full_name, bottle_hash, dry_run:)
       when "darwin"
         os_version ||= "macOS #{MacOS::Version.from_symbol(bottle_tag)}"
       when "linux"
-        os_version = (os_version || "Ubuntu 16.04.7").delete_suffix " LTS"
-        glibc_version = (tab["built_on"]["glibc_version"] if tab["built_on"].present?) || OS::GLIBC_CI_VERSION
+        os_version = (os_version || OS::CI_OS_VERSION).delete_suffix " LTS"
```

Comment:
The os_version from the constant won’t have the suffix so maybe don’t try to delete_suffix on it?